collect_app/src/main/java/org/odk/collect/android/widgets/MediaWidget.java

```diff
@@ -1,15 +1,47 @@
 package org.odk.collect.android.widgets;
 
 import android.content.Context;
+import android.graphics.Color;
 import android.media.MediaPlayer;
 import android.support.annotation.CallSuper;
 
 import org.javarosa.form.api.FormEntryPrompt;
 
+import timber.log.Timber;
+
 public abstract class MediaWidget extends QuestionWidget {
 
+    private int playColor;
+
     public MediaWidget(Context context, FormEntryPrompt prompt) {
         super(context, prompt);
+
+        String imageURI = this instanceof SelectImageMapWidget ? null : prompt.getImageText();
```

Comment:
Not sure, but I think we’re using the convention that initialisms (like URI) are capitalized like uri or imageUri. I have an audio clip of Josh Bloch himself recommending this.